client/test/preview/unit/routes/digitaltwins/execute/PipelineUtils.test.ts

```diff
@@ -76,32 +73,79 @@ describe('PipelineUtils', () => {
     expect(setLogButtonDisabled).toHaveBeenCalledWith(false);
   });
 
-  it('fetches job logs', async () => {
-    const mockGetPipelineJobs = jest
-      .spyOn(gitlabInstance, 'getPipelineJobs')
-      .mockResolvedValue([
-        {
-          id: 1,
-          name: 'job1',
-          status: 'success',
-          stage: 'build',
-        } as unknown as JobSchema,
-      ]);
-
-    const mockGetJobTrace = jest
-      .spyOn(gitlabInstance, 'getJobTrace')
-      .mockResolvedValue('log1');
-
-    const result = await fetchJobLogs(gitlabInstance, pipelineId);
-
-    expect(mockGetPipelineJobs).toHaveBeenCalledWith(
-      gitlabInstance.projectId,
-      pipelineId,
-    );
-    expect(mockGetJobTrace).toHaveBeenCalledWith(gitlabInstance.projectId, 1);
-    expect(result).toEqual([{ jobName: 'job1', log: 'log1' }]);
-
-    mockGetPipelineJobs.mockRestore();
-    mockGetJobTrace.mockRestore();
+  describe('fetchJobLogs', () => {
+    it('fetches job logs', async () => {
+      const mockJob: Partial<JobSchema> = { id: 1, name: 'job1' };
+  
+      const getPipelineJobsMock = jest.spyOn(gitlabInstance, 'getPipelineJobs');
```

Comment:
The Jest [naming convention](https://jestjs.io/docs/mock-functions) is incosistent. Sometimes `mock` is used as a prefix and sometimes `Mock` as suffix. It is good to be consistent in the codebase. We are using `mock` as a prefix; please follow this convention.

Reply:
Please follow these conventions.

Summary of Naming Conventions:

* Prefix with mock: e.g., mockFetchData, mockHandleClick.
* Use spy for spies: e.g., consoleLogSpy, apiCallSpy.
* Indicate behavior: e.g., mockFetchSuccess, mockFetchFailure.
* Use on for event handlers: e.g., mockOnClick, mockOnSubmit.
* Use is or has for booleans: e.g., mockIsAuthenticated, mockHasPermission.